app/views/rubric_criteria/_criterion_editor.html.erb

```diff
@@ -55,7 +57,7 @@
   </div>
 
   <div class='rubric_criteria_levels'>
-    <%= render partial: 'rubric_criterion_levels',
+    <%= render partial: 'rubric_criteria/rubric_criterion_levels',
```

Comment:
Something just happened with GitHub. It is behaving weirdly. Yes, I got an error because it was looking for criteria/rubric_criterion_levels.